modules/nlu/src/views/full/common/TrainNow.tsx

```diff
@@ -21,10 +22,11 @@ const TrainNow: FC<{ api: NLUApi; eventBus: any }> = ({ api, eventBus }) => {
 
   useEffect(() => {
     eventBus.on('statusbar.event', event => {
-      if (event.type === 'nlu' && event.message === 'Training complete') {
+      if (event.type === 'nlu' && (event.message === 'Training complete' || event.message === 'Training not needed')) {
```

Comment:
Can you move those messages to constants ? It would be safer, in case someone edits it in one place to change the label but forgets the other one

Reply:
Sure! Where should the constant be declared?

Reply:
I'd suggest exporting the constant from the backend where the phrase is being used, then import it here

Reply:
:+1: